src/lib/components/Header/Header.svelte

```diff
@@ -46,6 +46,7 @@
     alt=""
     width={headerBgWidth}
     height={headerBgHeight}
+    neverUpscaleImage={false}
```

Comment:
Small nit: I'd prefer if we could avoid a double negative here; maybe `upscaleImage` instead? Not blocking if you'd rather leave it as is.

Reply:
How about `canUpscaleImage`? `upscaleImage={true}` would incorrectly indicate that the image is _always_ upscaled.